apps/emqx/src/bpapi/emqx_bpapi.erl

```diff
@@ -83,6 +84,21 @@ supported_version(Node, API) ->
 supported_version(API) ->
     ets:lookup_element(?TAB, {?multicall, API}, #?TAB.version).
 
+-spec supported_apis(node()) -> [{api(), api_version()}].
+supported_apis(Node) ->
+    try
+        %% Make dialyzer happy...
+        MS = erlang:make_tuple(
+            record_info(size, ?TAB),
+            '_',
+            [{1, ?TAB}, {#?TAB.key, {Node, '$1'}}, {#?TAB.version, '$2'}]
+        ),
```

Comment:
I'd prefer to do something like:

```
-record(?TAB, {foo, bar, ...}).

-type ?TAB() :: #?TAB{foo :: ..., }.
```
if possible.

Reply:
:checkered_flag: